src/services/WolfDen.Application/Requests/Queries/Employees/EmployeeDirectory/GetAllEmployeeQueryHandler.cs

```diff
@@ -50,8 +52,17 @@ public async Task<List<EmployeeDirectoryDTO>> Handle(GetAllEmployeeQuery request
                    IsActive = e.IsActive
                })
             .ToListAsync(cancellationToken);
+            int totalpage=employees.Count;
```

Comment:
use valid spaces and indentations wherever required.

Reply:
okay